bot/exts/moderation/mod_utils.py

```diff
@@ -0,0 +1,30 @@
+from bot.bot import Bot
+from bot.constants import Roles
+from discord import Forbidden, Member
+from discord.ext.commands import Cog, Context, command, has_any_role
+from loguru import logger
+
+
+class ModUtils(Cog):
+    """Cog used for various utility moderation commands."""
```

Comment:
```suggestion
    """Cog used for various moderation utility commands."""
```
I feel like this sounds better.